src/components/v2/appDetails/k8Resource/nodeType/Node.component.tsx

```diff
@@ -255,14 +254,24 @@ function NodeComponent({ handleFocusTabs, externalLinks, monitoringTools, isDevt
                                     )}
                                     <div>
                                         <div>{node.name}</div>
-                                        <div
-                                            className={` app-summary__status-name f-${(
-                                                node?.status ||
-                                                node?.health?.status ||
-                                                ''
-                                            ).toLowerCase()}`}
-                                        >
-                                            {getNodeStatus(node)}
+                                        <div className="flex left">
+                                            <span
+                                                className={`mr-4 app-summary__status-name f-${(
+                                                    node?.status ||
+                                                    node?.health?.status ||
+                                                    ''
+                                                ).toLowerCase()}`}
+                                            >
+                                                {getNodeStatus(node)}
+                                            </span>
+                                            {getNodeMessage(node).length > 0 && (
```

Comment:
We can simple put a check like `node?.health?.message && <>{node.health.message.toLowerCase()}</> `

Reply:
I think we don't need the function here as it's performing the lowercase ops twice

Reply:
Done